examples/oim_loss.py

```diff
@@ -207,6 +209,8 @@ def adjust_lr(epoch):
     parser.add_argument('--evaluate', action='store_true',
                         help="evaluation only")
     parser.add_argument('--epochs', type=int, default=50)
+    parser.add_argument('--start_save', type=int, default=0,
+                        help="save checkpoints at specific epoch")
```

Comment:
Maybe it's better to say "start saving checkpoints after specific epoch"?

Reply:
Yes！It's more clear！

Reply:
Could you please change the help text when available and push the commit? This PR will automatically track the changes and I can then merge it directly. Thanks.

Reply:
OK.I'll do it tomorrow.

Reply:
Sure. No problem. Thanks a lot!

Reply:
Done!